examples/getting_started/pendulum.py

```diff
@@ -19,7 +21,61 @@
     InitialGuess,
     ObjectiveFcn,
     Objective,
+    PlotType,
+    ObjectiveList,
 )
+def plot_objectives(ocp):
+
+    # TODO : Add a label tag to CustomPlot for the legend on the graph
+
+    number_of_plots = 0
+    objective_names = []
+    same_objectives = [[], []]
+    number_of_same_objectives = 0
+    for nlp in ocp.nlp:
+        for j in nlp.J:
+            if j.name in objective_names:
+                same_objectives[0].append(objective_names.index(j.name))
+                same_objectives[1].append(number_of_plots)
+                number_of_same_objectives += 1
+            else:
+                objective_names.append(j.name)
+            number_of_plots += 1
+
+    step_size = 1 / (number_of_plots - number_of_same_objectives)
+    color = []
+    unique_color = 0
+    for i in range(number_of_plots):
+        if i in same_objectives[1]:
+            color += [plt.cm.viridis(step_size * same_objectives[0][same_objectives[1].index(i)])]
+        else:
+            color += [plt.cm.viridis(step_size * unique_color)]
+            unique_color += 1
+
+    def plot_obj(x, u, p, j):
+        plot_returned = cas.DM()
+        for i in range(np.shape(x)[1]):
+            if j.target is not None:
+                Plot = j.weighted_function(x[:, i], u, p, cas.DM(j.weight), cas.DM(j.target), cas.DM(dt))
```

Comment:
Please use `snake_case` for instances of classes (and `Camel` case for classes). Here `Plot` seems to define a class but is definitely a variable